src/cpp/src/utils/paged_attention_transformations.cpp

```diff
@@ -48,18 +47,19 @@ void set_kv_cache_type_and_shape(std::shared_ptr<ov::Model> model, DeviceConfig&
     OPENVINO_ASSERT(key_cache_params.size() == value_cache_params.size());
 
     size_t num_layers = key_cache_params.size();
-    // extract num_kv_heads and head_size
-    std::string key_cache_param_name = "key_cache.0";
+    // extract num_k_heads / num_v_heads and head_size
+    const std::string key_cache_param_name = "key_cache.0";
     OPENVINO_ASSERT(key_cache_params.count(key_cache_param_name) != 0, "key_cache.0 tensor not found among model parameters");
     ov::PartialShape k_shape = key_cache_params[key_cache_param_name]->get_partial_shape();
     OPENVINO_ASSERT(k_shape.rank().get_length() == 3, "KV cache shape is expected to have rank 3, while shape is ", k_shape);
     size_t head_size = k_shape[2].get_length();
```

Comment:
in case minicpm3 we also need (approach with calculation key_head_size != hidden_size / key_num_heads (it has different calculation formula)
```suggestion
    size_t k_head_size = k_shape[2].get_length();
    size_t v_head_size = v_shape[2].get_length();
```
is it planned to be covered in the same PR?